docs/fedora-releases-guide/what-to-know.md

```diff
@@ -0,0 +1,112 @@
+---
+title: What maintainers should know before adopting Packit for downstream release automation 
+sidebar_position: 5
+---
+# What maintainers should know before adopting Packit for downstream release automation 
+
+## Responsibilities
+
+Packit is designed to **assist** maintainers, not **replace** them.
+
+Maintainers retain key responsibilities:
+- reviewing and merging Packit-created pull requests
+- maintaining package quality
+- ensuring updates don't break dependant packages (Packit does not have yet a reverse dependencies check)
+
+:::tip `version_update_mask`
+
+The default Packit onboarding configuration enables automated releases only for `rawhide`. 
+
+When extending automation to branched Fedoras and EPELs, consider using [`version_update_mask`](https://packit.dev/docs/configuration#version_update_mask) to prevent breaking changes in stable Fedora versions.
+
+:::
+
+:::warning Packit pull requests can spam other maintainers
+
+SPam -> ask more about the spam
+
+:::
+
+:::warning missing notifications
+
+Packit bot user creates pull requests, Koji builds, and Bodhi updates, making it the default notification recipient.
+
+To receive pull request comment notifications, configure notification settings accordingly.
+
+There is an [open issue](https://github.com/packit/packit-service/issues/2404) for Bodhi update notifications.
+
+:::
+
+## Monitoring and observability
+
+Track *Packit Service* activity through:
+- the pull request updates and 
+- the [Packit Dashboard](https://dashboard.packit.dev/).
+
+Packit Service responds with 👀 for every valid command in pull requests. Missing eyes indicate:
```

Comment:
This is about downstream automation and in Pagure PRs Packit reacts with comments, not emojis.

Reply:
Oh right, I will fix the sentence.